src/Control/Session.php

```diff
@@ -167,6 +168,8 @@ public function __construct($data)
 
     /**
      * Init this session instance before usage
+     *
+     * @param HTTPRequest $request
```

Comment:
Redundant, IMO. Don't Repeat Yourself would advocate removing this when the type hints are already provided in-code.